passes/cmds/show.cc

```diff
@@ -758,7 +758,7 @@ struct ShowPass : public Pass {
 		std::vector<std::pair<std::string, RTLIL::Selection>> color_selections;
 		std::vector<std::pair<std::string, RTLIL::Selection>> label_selections;
 
-#if defined(_WIN32) || defined(YOSYS_DISABLE_SPAWN)
```

Comment:
I don't think we want to remove this one conditional.

Reply:
Fixed